### DISCUSSION TITLE
EC2 reset (volume delete) [Sep 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6369] [Discord Threads]
> 🙋‍♂️ **prasanth_23173**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Did you configure NC_DB for your docker setup? NocoDB uses this configured database for storing its meta data. If you had not configured this- it would have used SQLite file to create and manage meta data associated with NocoDB. And with your EC2 volume reset, you would have lost this.   
####    
#### You can start a fresh instance and try it out. Also 0.92.3 is very older version. Recommend to switch over to latest release.  
#  
> 🙋‍♂️ **prasanth_23173**  
#### Thanks for the reply, yeah i am aware that currently i am using very old version.    
#### i am also thinking of switching to newer version i feel it would be better to update to a stable version as some bug fixes are ongoing in the current 0.109 version series.   
####    
#### can you confirm the below?   
#### 1. i can directly update image from 0.92.3 to 109.6 without any additional change in my docker compose setup. i am just worried that i might lose some data if something is wrong with this much version update.   
#### 2. lets say if i get some issue can i revert back to 0.92.3 using same db? because as newer tables would have also will be migrated to db once i updated the version i want to know if it is backward compatible..  
#  
> 👨‍🦲 **dstala**  
#### Connecting to external DB- you shouldn't feel any difference. You can try out new version independently & then consider upgrading  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1137678799283167282)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
